test/src/test_error_injection.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+#include <string.h>
+#include <unistd.h>
+#include <postgres.h>
+#include <fmgr.h>
+#include <utils/builtins.h>
```

Comment:
No point IMO in having a separate file, especially after the rename to the `debug_point` stuff. Can add to that same file.

Reply:
Do you mean `test_debug_point.c`?

Reply:
yes